lib/jekyll/tags/highlight.rb

```diff
@@ -122,10 +120,10 @@ def render_codehighlighter(code)
 
       def add_code_tag(code)
         code_attributes = [
-          "class=\"language-#{@lang.to_s.tr("+", "-")}\"",
-          "data-lang=\"#{@lang}\"",
+          %(class="language-#{@lang.to_s.tr("+", "-")}"),
+          %(data-lang="#{@lang}"),
```

Comment:
One small suggestion: we could put everything in one line to avoid having to go through an array and `join` its contents: 
```ruby
code_attributes = %(class="language-#{@lang.to_s.tr("+", "-")}" data-lang="#{@lang}")
```

Reply:
Theoretically, we could have this in one line. But I think that would increase the line-length greater than 100 chars (as enforced by our rubocop config. Regardless, I cannot test this myself at the moment).

Reply:
Just tested it and it fits under the 100 char length limit :)